server.go

```diff
@@ -2853,6 +2849,7 @@ func parseHexColor(colorStr string) (color.RGBA, error) {
 func computeNextBackoff(currBackoff time.Duration) time.Duration {
 	// Double the current backoff, truncating if it exceeds our maximum.
 	nextBackoff := 2 * currBackoff
+	maximumBackoff := cfg.MaxBackoff
```

Comment:
Nit: no need to redefine this here, can just check `nextBackoff > cfg.MaxBackoff` below.

Reply:
@wpaulino You're right, fixed.